src/foam/u2/mlang/Demo.js

```diff
@@ -0,0 +1,90 @@
+foam.CLASS({
```

Comment:
Can/should we put this somewhere under the demo package with the rest of the demos rather than in the main code base?

Reply:
I moved it to the foam/demos package.